R/panels.R

```diff
@@ -183,9 +183,6 @@ panel_employment <- function() {
           inputId = "empOccSource",
           tags$ol(
             tags$li("Figures are for all age groups."),
-            tags$li("LSIP area totals are calculated by adding up the relevant local authorities, which may lead to rounding errors and differences compared to equivalent LEPs."),
-            tags$li("Only the top 5 occupations are shown for LSIP areas."),
-            tags$li("Percentages are based on proportion of total employment in LEP/LSIP."),
```

Comment:
Has this point on percentages changed?

Reply:
Yeah this is now the % of those with occupation matched to them. I left it out because now the % is the % of the column in the table, so it makes more sense in the context of the table (if not in reality)